mono/metadata/sre-encode.c

```diff
@@ -36,7 +36,7 @@ static guint32 create_typespec (MonoDynamicImage *assembly, MonoType *type);
 static void    encode_type (MonoDynamicImage *assembly, MonoType *type, SigBuffer *buf);
 static guint32 mono_image_typedef_or_ref (MonoDynamicImage *assembly, MonoType *type);
 
-static guint32
+static G_GNUC_UNUSED guint32
```

Comment:
Can you just put the whole function between `#if G_BYTE_ORDER != G_LITTLE_ENDIAN` ?